packages/framework/fluid-framework/src/index.ts

```diff
@@ -36,6 +36,9 @@ export {
 	DOProviderContainerRuntimeFactory,
 	FluidContainer,
 	ServiceAudience,
+	createDOProviderContainerRuntimeFactory,
+	createFluidContainer,
+	createServiceAudience,
```

Comment:
These shouldn't be re-exported from fluid-framework (the classes shouldn't be exported either).  They are internal implementation details that customers shouldn't use directly.

In general - `fluid-framework` should be 100% stuff we want customers to use directly.